src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__title {
+    color: $color-main;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.02);
```

Comment:
The hover effect currently scales the card by 1.02. Ensure that this meets the task requirement for the card hover effect. If the task specifies a different scale, adjust this value accordingly.